zcash_client_sqlite/src/wallet.rs

```diff
@@ -173,6 +174,36 @@ pub fn get_address<P: consensus::Parameters>(
         })
 }
 
+pub(crate) fn get_address_ua<P: consensus::Parameters>(
+    wdb: &WalletDb<P>,
+    account: AccountId,
+) -> Result<Option<UnifiedAddress>, SqliteClientError> {
+    // This returns the first diversified address, which will be the default one.
+    let addr: Option<String> = wdb
+        .conn
+        .query_row(
+            "SELECT address FROM addresses WHERE account = :account",
```

Comment:
```suggestion
            "SELECT address FROM addresses WHERE account = :account ORDER BY diversifier_index LIMIT 1",
```
Please make the "first diversified address" condition explicit.

Reply:
In a pairing we established that this didn't work, because diversifier indices are stored as `BLOB` types, and ordering there would not order numerically. As it is, this method will always return a single address because we don't expose a way to derive more, and when we do this method will need to take a diversifier index.